modules/effects/src/index.ts

```diff
@@ -11,10 +11,15 @@ export { EffectNotification } from './effect_notification';
 export {
   ROOT_EFFECTS_INIT,
   rootEffectsInit,
+  EffectsRootModule,
```

Comment:
@brandonroberts The github merge conflict thing is definitely weird. As now apparently i've added this, even though it said this is in master.